test/Gridify.Elasticsearch.Tests/GridifyExtensionsTests.cs

```diff
@@ -0,0 +1,597 @@
+﻿using System;
+using Elastic.Clients.Elasticsearch;
+using Elastic.Transport.Extensions;
+using Xunit;
+
+namespace Gridify.Elasticsearch.Tests;
+
+public class GridifyExtensionsTests
+{
+   private readonly ElasticsearchClient _client = new();
+
+   public GridifyExtensionsTests()
+   {
+      // Disable Elasticsearch naming policy and use property names as they are
+      GridifyGlobalConfiguration.CustomElasticsearchNamingAction = p => p;
```

Comment:
Parsing decimal and double values is a Culture-specific operation, for example, on my machine, 18 tests failed because my decimal separator is a `,`, this is not a solution to fix this problem in Gridify, but for now add this to your tests to fix the tests in everyone's machines. 
``` csharp
GridifyGlobalConfiguration.CustomElasticsearchNamingAction = p => p;
CultureInfo.CurrentCulture = CultureInfo.InvariantCulture; // <--
```